### ISSUE TITLE
Discrepancy between number of stars

### ISSUE THREAD

**Issue by jamiewilson** [algolia/github-awesome-autocomplete#31]
I noticed that there's always a difference between the number of stars a repo has in the search results and the number of stars displayed on the repo itself. I'm guessing that the results don't reflect Unstars or something like that, cause it's always more than the actual count.

![stars-discrepancy](https://cloud.githubusercontent.com/assets/3308673/13618789/b07f73da-e54b-11e5-878d-766f73925d6d.png)

**Comment by redox** [comment]
@jamiewilson you're right, we don't collect Unstars for now; not sure that explains it all because the diff seems pretty high :/ Need to dig more into githubarchives.org data to confirm.

**Comment by jamiewilson** [comment]
Cool. For now though, I'll just assume that the search results number is the correct one. It gave some of my projects a pretty nice bump :)

**Comment by redox** [comment]
@jamiewilson ahah indeed :) We use the number of stars for relevance purpose, so it's not a big deal if the values are not super up-to-date but better if we can ensure we don't have too much difference.
